packages/analytics-js-integrations/src/integrations/Amplitude/browser.js

```diff
@@ -85,14 +85,23 @@ class Amplitude {
 
     // rudderElement.message.context will always be present as part of identify event payload.
     const { traits } = rudderElement.message.context;
-    const { userId } = rudderElement.message;
-
+    const { userId, integrations } = rudderElement.message;
```

Comment:
don't have common library to share functionality between cloud and device mode code flows?

Reply:
No and also there is a difference how unset is done in cloud vs js-sdk